### ISSUE TITLE
Make a version that works agains future clair v3

### ISSUE THREAD

**Issue by arminc** [arminc/clair-scanner#27]

**Comment by dzwicker** [comment]
v1 is gone in the clair repository. v3 is only present

**Comment by aurcioli-handy** [comment]
Yeah.. can't compile b/c Stderr: clair.go:10:2: cannot find package "github.com/coreos/clair/api/v1" in any of:

**Comment by gugahoi** [comment]
Still having the same problem:

```bash
⌬ go get -u -d github.com/arminc/clair-scanner
package github.com/coreos/clair/api/v1: cannot find package "github.com/coreos/clair/api/v1" in any of:
	/usr/local/go/src/github.com/coreos/clair/api/v1 (from $GOROOT)
	/Users/p780680/go/src/github.com/coreos/clair/api/v1 (from $GOPATH)
```

how can I compile from src?

**Comment by Ajitesh2012** [comment]
Hello Team,

I am also facing the same issue with clair scanner. Its is pointing api/v1 which is in master branch. is there way to route to either of the stable release which has api/v1.

![image](https://user-images.githubusercontent.com/47196587/63032825-78b18e80-beae-11e9-8253-49a09be131d2.png)

Also make sure && and make build is also stuck. 

Link: https://gist.github.com/sdorsett/1fd15bc0a65a8ec1b74b6a22fa37910a#file-notes-txt-L8

![image](https://user-images.githubusercontent.com/47196587/63033000-d04ffa00-beae-11e9-8feb-6d15c7cd7b32.png)
